test/test_helper.rb

```diff
@@ -24,6 +24,7 @@
 require 'mocha/setup'
 
 require 'sucker_punch/testing/inline'
+require 'pry'
```

Comment:
Benchmark how much boot time this adds then

On Aug 1, 2016 13:28, "Adam Panzer" <notifications@github.com> wrote:

> In test/test_helper.rb
> <https://github.com/zendesk/samson/pull/1163#discussion_r73046938>:
>
> > @@ -24,6 +24,7 @@
> >  require 'mocha/setup'
> >
> >  require 'sucker_punch/testing/inline'
> > +require 'pry'
>
> Why? Pry is useful in test env
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1163/files/de1eac82baece3401cd32f0072a54a59ad332c96#r73046938>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ4at_ZuloKkHqIRbXHCfl3pntebGks5qblbqgaJpZM4JVi_g>
> .
>